### ISSUE TITLE
The name 'Edmx' does not exist in the current context

### ISSUE THREAD

**Issue by LarsBauer** [OData/ODataConnectedService#151]
### Describe the bug
When generating model classes for the Web API of [Common Data Service](https://docs.microsoft.com/en-us/dynamics365/customer-engagement/web-api/about?view=dynamics-ce-odata-9) I get this error message in `Reference.cs`. I am not sure if this issue is related to CDS but it can be reproduced reliably when generating for CDS. Here is the relevant code snippet:

```csharp
[global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.OData.Client.Design.T4", "#VersionNumber#")]
private static global::Microsoft.OData.Edm.IEdmModel LoadModelFromString()
{
    global::System.Xml.XmlReader reader = CreateXmlReader(Edmx); // <= Edmx does not exist
    try
    {
        return global::Microsoft.OData.Edm.Csdl.CsdlReader.Parse(reader, getReferencedModelFromMap);
    }
    finally
    {
        ((global::System.IDisposable)(reader)).Dispose();
    }
}
```

#### Version of the Project affected
ODataConnectedService v0.10.0
VisualStudio Version v16.6.2
Microsoft.Odata.Client v7.6.4

#### To Reproduce
1. Generate model classes for Common Data Service Web API
2. Try to rebuild solution
3. The described error is displayed in the Visual Studio error list

#### Expected behavior
The extension should generate working code.

#### Actual behavior
The code generated by the extension does not compile.

**Comment by Sreejithpin** [comment]
Hi,
 We are investigating on this. When we try the same with services, we are not seeing this pattern. Can you please explain the steps you are going through when generating code from connected services. Also can you please validated you are using the 0.10.0 version

**Comment by LarsBauer** [comment]
Hi,
Thanks for the quick answer.
Yes, I am using the latest version (0.10.0). Here is the configuration I provided. Nothing special from my point of view.
![odata_configure_endpoint](https://user-images.githubusercontent.com/3920045/85673473-34913c00-b6c4-11ea-9f94-431a1451c03f.png)
![odata_settings](https://user-images.githubusercontent.com/3920045/85673490-3955f000-b6c4-11ea-8277-6386de9ae21f.png)

I just realized there are some missing types as well so maybe there is some kind of error in the metadata document of the service itself?

Error message | File
-----------------|-----
The type or namespace name 'ExportTranslationResponse' does not exist in the namespace 'Microsoft.Dynamics.CRM' | ExtensionMethods.cs
The type or namespace name 'RetrieveTimelineWallRecordsResponse' does not exist in the namespace 'Microsoft.Dynamics.CRM' | ExtensionMethods.cs
The type or namespace name 'CreateExceptionResponse' does not exist in the namespace 'Microsoft.Dynamics.CRM' | ExtensionMethods.cs
The type or namespace name 'ServicePlan' does not exist in the namespace 'Microsoft.Dynamics.CRM' | UserLicenseInfo.cs

Here is the `metadata.xml` of a practically vanilla environment:
[metadata.zip](https://github.com/OData/ODataConnectedService/files/4829639/metadata.zip)
